entry/render/index.js

```diff
@@ -1,16 +1,18 @@
 import serializeJavascript from 'serialize-javascript';
 import makeExtractor from '../makeExtractor';
 import clientContextKey from '../clientContextKey';
+import createCSPHandler from '../csp';
 
 import render from '__sku_alias__renderEntry';
 
 const libraryName = SKU_LIBRARY_NAME;
 const publicPath = __SKU_PUBLIC_PATH__;
+const csp = __SKU_CSP__;
 
 export const serializeConfig = (config) =>
-  `<script>window.${clientContextKey} = ${serializeJavascript(
+  `<script id="${clientContextKey}" type="application/json">${serializeJavascript(
```

Comment:
Now that this is JSON rather than JavaScript, might need to be mindful that [serialize-javascript](https://www.npmjs.com/package/serialize-javascript) supports a superset of JSON, so we might need to ensure that only strict JSON is allowed here.